src/Select/index.js

```diff
@@ -67,10 +80,9 @@ type SelectPropsT = {
   /** Placeholder text for the form control. */
   placeholder?: string,
   /** Disables the form control. */
-  disabled?: boolean,
-  /** Makes a cssOnly select. */
-  cssOnly?: boolean
```

Comment:
RIP

Reply:
Hahaha. I could tell by your cod examples you loves them. Shouldn’t be hard to make a wrapper for.

Reply:
It's more that menu has issues when inside something that is transformed or has overflow hidden. Which is like 3/4 of my mobile interface.